src/components/Form/DeleteDocumentForm.jsx

```diff
@@ -50,30 +55,29 @@ const DeleteDocumentForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
-  /* eslint-disable jsx-a11y/label-has-associated-control */
   return (
     <FormSection
       title="Delete Document"
       state={state}
       statusType="Deletion status"
       defaultMessage="To be deleted..."
     >
-      <form onSubmit={handleDeleteDocument} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="delete-doctype">Select document type to delete: </label>
-          <DocumentSelection htmlId="delete-doctype" />{' '}
-          <button disabled={state.processing} type="submit">
-            Delete Document
-          </button>
-        </div>
-      </form>
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleDeleteDocument}>
+          <InputLabel id="delete-doctype">
```

Comment:
Already tested, this `<InputLabel>`'s `id` needs to be `htmlFor` to get rid of blue warnings in dev tools. Currently, it is not linked to what it's trying to label.

Reply:
Getting rid of the incorrect labels `id` with the correct `htmlFor` attribute.